CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ option( BUILD_SHARED_LIBS             "Build exiv2lib as a shared library"
 option( EXIV2_ENABLE_XMP              "Build with XMP metadata support"                       ON  )
 option( EXIV2_ENABLE_EXTERNAL_XMP     "Use external version of XMP"                           OFF )
 option( EXIV2_ENABLE_PNG              "Build with png support (requires libz)"                ON  )
+option( EXIV2_ENABLE_HEIF             "Build with HEIF support (requires libheif)"            OFF )
```

Comment:
I checked out your branch and I tried to enable this option without actually installing the library. The CMake configuration succeeded and when I tried to compile I got a compilation error:

```
In file included from /media/linuxDev/programming/exiv2/src/image.cpp:38:
/media/linuxDev/programming/exiv2/src/heifimage_int.hpp:29:10: fatal error: libheif/heif.h: No such file or directory
 #include <libheif/heif.h>
          ^~~~~~~~~~~~~~~~
```

Somehow the `FindHEIF.cmake` you are providing would need to detect if the library/headers have been detected properly.